### PR TITLE
Import / export csv file endpoints

### PR DESCRIPTION
**Pull request by n0rdy**

### CODE REVIEW

File: registry-manager-service-api/src/main/java/de/adorsys/registry/manager/service/exception/IbanException.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package de.adorsys.registry.manager.service.exception;
+
+public class IbanException extends RuntimeException {
```
Comment (spetrychenko) [inline registry-manager-service-api/src/main/java/de/adorsys/registry/manager/service/exception/IbanException.java:3]: have you added this exception to the advisor?

File: registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/impl/AspspCsvServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                             .schemaFor(AspspCsvRecord.class)
+                             .rebuild()
+                             .replaceColumn(nameColumnIndex, new CsvSchema.Column(nameColumnIndex, ASPSP_NAME_COLUMN_NAME))
+                             .build();
+        }
+
+        try {
+            return mapper.writer(schema).writeValueAsString(aspsp);
+        } catch (JsonProcessingException e) {
+            logger.warn("Exception occurred while indexes were being written into a CSV: {}", e.getMessage());
```
Comment (spetrychenko) [inline registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/impl/AspspCsvServiceImpl.java]: this is anti-pattern `log and re-throw`, just re-throw with given message

File: registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/impl/AspspServiceImpl.java [hunk 3]
```diff
@@ -24,6 +27,37 @@ public AspspServiceImpl(AspspRepository repository, AspspBOConverter converter)
...
+    @Override
+    public List<AspspBO> getByIban(String iban, int page, int size) {
+        logger.info("Trying to get ASPSPs by IBAN {}", iban);
+
+        String bankCode;
+
+        try {
+            bankCode = Iban.valueOf(iban).getBankCode();
+        } catch (Iban4jException e) {
+            logger.error(iban, e);
```
Comment (spetrychenko) [inline registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/impl/AspspServiceImpl.java]: just re-throw with a corresponding message

File: registry-manager-service-impl/pom.xml [hunk 4]
```diff
@@ -48,6 +48,17 @@
             <artifactId>mapstruct-jdk8</artifactId>
```
Comment (spetrychenko) [inline registry-manager-service-impl/pom.xml]: could you be so kind and update mapstruct version to the latest one

File: registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/impl/AspspCsvServiceImpl.java (hunk 2 above)
Comment (n0rdy) [reply registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/impl/AspspCsvServiceImpl.java]: my bad - was not that attentive while copy-pasting code from the adapter app.

File: registry-manager-service-api/src/main/java/de/adorsys/registry/manager/service/exception/IbanException.java (hunk 1 above)
Comment (n0rdy) [reply registry-manager-service-api/src/main/java/de/adorsys/registry/manager/service/exception/IbanException.java:3]: I know. I want to create a separate task for adjusting the exception handling.

File: registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/impl/AspspCsvServiceImpl.java (hunk 2 above)
Comment (n0rdy) [reply registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/impl/AspspCsvServiceImpl.java]: Done, thanks

File: registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/impl/AspspServiceImpl.java (hunk 3 above)
Comment (n0rdy) [reply registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/impl/AspspServiceImpl.java]: Done, thanks

File: registry-manager-service-impl/pom.xml (hunk 4 above)
Comment (n0rdy) [reply registry-manager-service-impl/pom.xml]: Done

File: registry-manager-service-api/src/main/java/de/adorsys/registry/manager/service/exception/IbanException.java (hunk 1 above)
Comment (spetrychenko) [reply registry-manager-service-api/src/main/java/de/adorsys/registry/manager/service/exception/IbanException.java:3]: ok, just do not forget about that

**Review by spetrychenko — APPROVED** [review]

**Pull request merged**